bika/lims/tests/doctests/RemoveAnalysesFromAnalysisRequest.rst

```diff
@@ -148,8 +148,30 @@ transitions to "to_be_verified" because follows `Fe`:
     >>> api.get_workflow_status_of(ar)
     'to_be_verified'
 
-And if we remove analysis `Fe` (in `to_be_verified` state), the Analysis Request
-will follow `Au` analysis (that is `verified`):
+Analyses which are in the state `to_be_verified` can **not** be removed.
+Therefore, if we try to remove the analysis `Fe` (in `to_be_verified` state),
+the Analysis Request will stay in `to_be_verified` and the Analysis will still
+be assigned:
+
+    >>> new_analyses = ar.setAnalyses([Au])
+
+    >>> analysis_fe in ar.objectValues()
```

Comment:
Just remember that this won't work with Partitions. In 99% of cases (except in `ARAnalysesField` and maybe in 1 or 2 other places) use `ar.getAnalyses(full_objects=True)`.